js/core/include-config.js

```diff
@@ -21,7 +21,7 @@ define(
           }
           return val;
         }
-        script.innerText = JSON.stringify(conf, confFilter, 2);
+        script.innerHTML = JSON.stringify(conf.initialUserConfig, confFilter, 2);
```

Comment:
Note, innerText doesn't work... it doesn't output when respec calls .innerHTML on the root node.